sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -541,7 +541,8 @@ class ResourceTypes(object):
     def __init__(self, **kwargs) -> None:
```

Comment:
`container` is missing in the docstring

Reply:
Also maybe silly question, but how does a `container` resource type work with tables? Is there a container concept in tables, or is the expectation that you could create the sas in this library and use it across storage libraries?

Reply:
- Actually the term `container` here is a litter confused for me, from the test result it's the accessibility required for creating Tables.
- The sas token generated here can also be used in other libraries that want to access the same storage account.

Reply:
> `container` is missing in the docstring

Added, thanks for catching!